src/common/lib/utils.ts

```diff
@@ -170,6 +172,21 @@ const utils = {
         });
     });
   },
+  getBoostagramFromInvoice: (
```

Comment:
the parameter that is passed in here is a custom_record object? 
If so I'd rename that function.

Reply:
Let's zoom